README.md

```diff
@@ -1,23 +1,23 @@
 <p align="center">
-  <a target="_blank" href="https://transform.co/metricflow">
+  <a target="_blank" href="https://www.getdbt.com/">
```

Comment:
Do we want to change this yet? I think we'd want this to continue linking to wherever the MetricFlow docs are housed.

Ideally we'd get things merged in under https://www.getdbt.com/product/semantic-layer/ or something, but that doesn't seem to have happened yet.

Reply:
Fair callout - I'll remove this and we can update it once we've updated the top level org links.